src/serac/physics/tests/solid.cpp

```diff
@@ -320,14 +499,16 @@ void functional_parameterized_solid_test(double expected_disp_norm)
   EXPECT_NEAR(expected_disp_norm, norm(solid_solver.displacement()), 1.0e-6);
 }
 
-TEST(SolidMechanics, 2DQuadParameterizedStatic) { functional_parameterized_solid_test<2, 2>(2.1906312704664623); }
+// TEST(SolidMechanics, 2DQuadParameterizedStatic) { functional_parameterized_solid_test<2, 2>(2.1906312704664623); }
 
-TEST(SolidMechanics, 3DQuadStaticJ2) { functional_solid_test_static_J2(); }
+// TEST(SolidMechanics, 3DQuadStaticJ2) { functional_solid_test_static_J2(); }
 
-TEST(SolidMechanics, 2DLinearPressure)
-{
-  functional_solid_test_boundary<1, 2>(0.028525698834671667, TestType::Pressure);
-}
+TEST(SolidMechanics, SpatialBoundaryCondition) { functional_solid_spatial_essential_bc(); }
+
+// TEST(SolidMechanics, 2DLinearPressure)
+//{
+//  functional_solid_test_boundary<1, 2>(0.028525698834671667, TestType::Pressure);
+//}
```

Comment:
can these tests be re-enabled?

Reply:
Oh yes - this was a shortcut for testing.